usaspending_api/database_scripts/broker_matviews/agency_lookup_matview.sql

```diff
@@ -0,0 +1,25 @@
+drop materialized view if exists agency_lookup;
+
+create materialized view agency_lookup as
```

Comment:
very minor: For the existing "API" matview SQL scripts all keywords are uppercase. Postgres doesn't care. We should keep it consistent as much as we can.

Reply:
We can alter this. Lowercase was just a personal preference but Postgres docs also use uppercase so we should be consistent with that in addition to how we're doing it in other areas of our codebase.